### PR TITLE
oneVideo Adapter - Dynamic TTL support (SAPR-15473)

### PR DESCRIPTION
**Pull request by adam-browning**
## Type of change
- [ x] Feature

## Description of change
oneVideoBidAdapter now supports a new parameter to set a the Time To Live value dynamically.
Default has increased fron 100 to 300 seconds
To pass custom ttl use `bid.params.video.ttl = 300`
type: Number
Acceptable value range: 1 - 3600 (seconds) - exceeding values will default to 300

- test parameters for validating bids
note: This is using e2etest mode 
```
var adUnits = [
    {
        code: 'video1',
          mediaTypes: {
            video: {
                  context: 'outstream',
                  playerSize: [480, 640]
            }
          },
          bids: [
            {
              bidder: 'oneVideo',
              params: {
                video: {
                  playerWidth: 480,
                  playerHeight: 640,
                  mimes: ['video/mp4', 'application/javascript'],
                  api: [2],
                  e2etest: true,
                  ttl: 500
                },
                pubId: 'Whatever'
                }
            }
          ]
      }
  ]
```
## Other information
Hi @prasad457, could you please review? 
Thanks in advance,
Adam

### CODE REVIEW

**Review by kprasadpvv — APPROVED** [review]

**Review by kprasadpvv — APPROVED** [review]

File: test/spec/modules/liveIntentIdSystem_spec.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const PUBLISHER_ID = '89899';
 const defaultConfigParams = { params: {publisherId: PUBLISHER_ID} };
 const responseHeader = {'Content-Type': 'application/json'}
 
-describe('LiveIntentId', function() {
+xdescribe('LiveIntentId', function() {
```
Comment (kprasadpvv) [inline test/spec/modules/liveIntentIdSystem_spec.js:10]: `xdescribe`

Comment (adam-browning) [reply test/spec/modules/liveIntentIdSystem_spec.js]: removed 3p unit test prevention

**Review by kprasadpvv — APPROVED** [review]

File: modules/oneVideoBidAdapter.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ Connects to Verizon Media's Video SSP (AKA ONE Video / Adap.tv) demand source to
                   inventoryid: 123,
                   minduration: 10,
                   maxduration: 30,
+                  ttl: 300
```
Comment (kprasadpvv) [inline modules/oneVideoBidAdapter.md:43]: seems like you are missing comma `,` after `ttl: 300`

Comment (adam-browning) [reply modules/oneVideoBidAdapter.md]: thanks for catching :) fixed accordingly

**Review by kprasadpvv — APPROVED** [review]

**Comment by adam-browning** [comment]
Hello @msm0504 
I believe that circleci is failing due to pubGENIUS adapter
I could not find any errors regarding our oneVideo adapter
Thanks in advance,
Adam

**Review by msm0504 — APPROVED** [review]

**Comment by adam-browning** [comment]
Hello @jsnellbaker, 
AFAIK CircleCi is failing here due unit test issues with the following adapters.
- LiveIntentId
- pubGENIUS adapter
I have updated upstream

**Comment by msm0504** [comment]
@adam-browning Can you try pulling the latest changes from Prebid master into your branch? Maybe that will fix the failing tests.

**Comment by adam-browning** [comment]
Hi @msm0504 I have been trying that for the past few days.
Tried again not and still failing.
16 Unit tests are failing for LiveIntentId adapter
image

**Pull request closed without merging**